openstack-config-sample.yaml

```diff
@@ -3,8 +3,6 @@ exporter:
   prometheus_port: 9102
 openstack:
   auth_url: https://identity-3.qa-de-1.cloud.sap/v3
-  username: <username here>
-  password: <password here>
```

Comment:
I'd just comment this out and leave a comment, that this still can be used, but `--secrets` is encouraged.